### PR TITLE
Add manual test for node delete

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/2186

### CODE REVIEW

File: docs/content/manual/release-specific/v1.1.1/delete-node.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+1. Disable Node Scheduling and set Eviction Requested to true from the browser.
+2. Taint node-1 with kubectl and wait for pods to re-deploy.
+    ```
+    kubectl taint nodes ${NODE} nodetype=storage:NoExecute
+    ```
+
+    2.1. Check longhorn pods are not scheduled to node-1.
+
+    2.2. Node status should be Down.
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.1.1/delete-node.md:24]: Node status should be Down on Longhorn UI - Node page.

File: docs/content/manual/release-specific/v1.1.1/delete-node.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+**When** node-1 deleted from the browser.
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.1.1/delete-node.md:26]: node-1 deleted from the Longhorn UI using the menu `remove node`.

File: docs/content/manual/release-specific/v1.1.1/delete-node.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    1.1. Check longhorn pods are not scheduled to node-1.
+
+    1.2. Node status should be Down.
+
+    1.3. Check longhorn replicas are running on node-1.
+    ```
+    kubectl -n longhorn-system get replicas
+    ```
+
+**When** node-1 deleted from the browser.
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.1.1/delete-node.md:58]: node-1 deleted from the Longhorn UI using the menu remove node.

**Comment by khushboo-rancher** [comment]
IN general LGTM, have put some comments.
Should we mention when the `remove node` should be greyed out or enabled?

**Review by innobead — APPROVED** [review]

**Pull request merged**